app/src/components/governance/PageProposal.vue

```diff
@@ -258,19 +141,38 @@ export default {
     noVotes() {
       return BigNumber(this.proposal.tally.total).eq(0)
     },
-    getNextProposalId() {
-      let id = this.getProposalIndex(-1)
-      return id
+    tallyHasValues() {
+      return Object.values(this.proposal.tally)
+        .filter((value) => value !== `Tally`)
+        .find((value) => value)
     },
-    getPrevProposalId() {
-      let id = this.getProposalIndex(1)
-      return id
+    isSecond() {
```

Comment:
this is confusing. can we find a better name?

Reply:
`isPolkadotDemocracy` I guess